content/tutorials/manage-the-rippled-server/installation/build-run-rippled-macos.md

```diff
@@ -24,7 +24,7 @@ For development purposes, run `rippled` as a non-admin user, not using `sudo`.
 
         $ brew install git cmake pkg-config protobuf openssl ninja
 
-0. Install Boost 1.70.0 or higher. `rippled` 1.4.0 is compatible with Boost 1.70.0 and higher. The latest version of Boost available in the Homebrew repositories isn't new enough, so you must install Boost manually. (The following examples use Boost 1.71.0, which was the newest version at the time of writing.)
+0. Install Boost 1.70.0 or higher. `rippled` 1.7.0 is compatible with Boost 1.70.0 and higher. The latest version of Boost available in the Homebrew repositories isn't new enough, so you must install Boost manually. (The following examples use Boost 1.71.0, which was the newest version at the time of writing.)
```

Comment:
Would it make sense to change the rippled version to a variable that gets the most recent version from the config?

Reply:
I considered this, but the details of this statement need to be revisited from release to release because Boost compatibility varies, so it wouldn't really save us much.